Firestore/core/test/firebase/firestore/remote/datastore_test.mm

```diff
@@ -105,7 +105,7 @@ void CancelLastCall() {
       : worker_queue{std::make_shared<AsyncQueue>(
             absl::make_unique<ExecutorLibdispatch>(dispatch_queue_create(
                 "datastore_test", DISPATCH_QUEUE_SERIAL)))},
-        database_info{DatabaseId{"p", "d"}, "", "some.host", false},
+        database_info{DatabaseId{"p", "d"}, "", "localhost", false},
```

Comment:
No objection, though this seems unrelated.

Reply:
Sorry this is unrelated but it was required to make the tests work on a plane :-D